tests/dedup_help.txt

```diff
@@ -0,0 +1,135 @@
+UMI-Tools: Version 1.1.4
```

Comment:
If I was being really nitpicky, I'd say including the version number in the test file is going to be a pain as it means updating with every new version.